bench_test.go

```diff
@@ -0,0 +1,114 @@
+package goyave
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"runtime"
+	"testing"
+
+	"github.com/System-Glitch/goyave/v2/config"
+)
+
+var staticGoyave *Router
+
+func calcMem(name string, load func()) {
+	m := new(runtime.MemStats)
+
+	// before
+	runtime.GC()
+	runtime.GC()
+	runtime.GC()
+	runtime.GC()
+	runtime.ReadMemStats(m)
+	before := m.HeapAlloc
+
+	load()
+
+	// after
+	runtime.GC()
+	runtime.GC()
+	runtime.GC()
+	runtime.GC()
```

Comment:
Why is quadruple manual garbage collection needed?

Reply:
Oh, I just want to make sure GC collects all the garbage

Reply:
I think it's not very clean. Is there a better way?

Reply:
At the moment, I don't have any other way